extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,241 @@
+          "title": "Property Name",
+          "type": "string",
+          "description": "Property Name Configured by Pixel, Analytics, Custom Feeds"
+        },
+        "xdm:clickThroughValue": {
+          "title": "Click Through Value",
+          "type": "number",
+          "description": "Raw Conversion Value if Triggering Event Type is Click"
+        },
+        "xdm:clickThroughCount": {
+          "title": "Click Through Count",
+          "type": "number",
+          "description": "Raw Conversion Count if Triggering Event Type is Click"
+        },
+        "xdm:viewThroughValue": {
+          "title": "View Through Value",
+          "type": "number",
+          "description": "Raw Conversion Value if Triggering Event Type is Impression"
+        },
+        "xdm:viewThroughCount": {
+          "title": "View Through Count",
+          "type": "number",
+          "description": "Raw Conversion Count if Triggering Event Type is Impression"
+        },
+        "xdm:clickviewThroughValue": {
+          "title": "Click View Through Value",
+          "type": "number",
+          "description": "Raw conversion value if trigger event is click but impressions present in path"
+        },
+        "xdm:clickviewThroughCount": {
+          "title": "Click View Through Count",
+          "type": "number",
+          "description": "Raw conversion Count if trigger event is click but impressions present in path"
+        },
+        "xdm:viewclickThroughValue": {
+          "title": "View Click Through Value",
+          "type": "number",
+          "description": "Raw conversion value if trigger event is impression but clicks present in path"
+        },
+        "xdm:viewclickThroughCount": {
+          "title": "View View Through Count",
+          "type": "number",
+          "description": "Raw conversion count if trigger event is impression but clicks present in path"
+        },
+        "xdm:pathSize": {
+          "title": "Path Size",
+          "type": "integer",
+          "description": "Path Size of Trigger Event Type. It is a numerical value that represents the count of the conversion path that triggered the event"
+        },
+        "xdm:conversionType": {
+          "title": "Conversion Type",
+          "type": "integer",
+          "description": "Feed Conversion Type indicates where the conversion event originated from - examples client, analytics, cross device"
+        },
+        "xdm:productId": {
+          "title": "Product Identifier",
+          "type": "string",
+          "description": "Product Id on the search advertising platform"
+        },
+        "xdm:device": {
+          "title": "Device",
+          "$ref": "https://ns.adobe.com/xdm/context/device",
+          "description": "The device from where the ad was displayed"
+        },
+        "xdm:eventUid": {
+          "title": "Event UID",
+          "type": "string",
+          "description": "Unique Identifier of the Event"
```

Comment:
in the description, can you add `Conversion`?
Unique Identifier of the `Conversion` Event

Reply:
Makes sense. Will fix